tests/frontDesk/clinicalExamination.js

```diff
@@ -42,35 +42,46 @@ step("put medications <prescriptionNames>", async function (prescriptionNames) {
 })
 
 step("Doctor prescribes medicines <prescriptionNames>", async function (prescriptionNames) {
-    var prescriptionFile = `./bahmni-e2e-common-flows/data/${prescriptionNames}.json`;
-    gauge.dataStore.scenarioStore.put("prescriptions", prescriptionFile)
-    var drugName = gauge.dataStore.scenarioStore.get("Drug Name")
-
-    var medicalPrescriptions = JSON.parse(fileExtension.parseContent(prescriptionFile))
-    gauge.message(medicalPrescriptions)
-
-    if (medicalPrescriptions.drug_name != null) {
-        if (drugName == null)
-            drugName = medicalPrescriptions.drug_name;
-        if (await textBox(toRightOf("Drug Name")).exists()) {
-            await write(drugName, into(textBox(toRightOf("Drug Name"))));
-            await click(link(drugName, below(textBox(toRightOf("Drug Name")))));
-            await dropDown(toRightOf("Units")).select(medicalPrescriptions.units);
-            await dropDown(toRightOf("Frequency")).select(medicalPrescriptions.frequency)
-            await write(medicalPrescriptions.dose, into(textBox(toRightOf("Dose"))));
-            await write(medicalPrescriptions.duration, into(textBox(toRightOf("Duration"))));
-        }
-        else {
-            await write(drugName, into(textBox(below("Drug Name"))));
-            await click(link(drugName, below("Drug Name")));
-            await dropDown(below("Units")).select(medicalPrescriptions.units);
-            await dropDown(below("Frequency")).select(medicalPrescriptions.frequency)
-            await write(medicalPrescriptions.dose, into(textBox(below("Dose"))));
-            await write(medicalPrescriptions.duration, into(textBox(below("Duration"))));
+    if (prescriptionNames.includes(',')) { var prescriptionsList = prescriptionNames.split(',') }
+    else { var prescriptionsList = prescriptionNames.split() }
+    console.log(prescriptionsList)
```

Comment:
We can directly use the below line "if else" is not needed
`var prescriptionsList = prescriptionNames.split(',')`

Reply:
remove console logs

Reply:
done